cmd/av/pr.go

```diff
@@ -63,15 +107,42 @@ var prCreateCmd = &cobra.Command{
 			prBaseBranch = defaultBranch
 		}
 
-		// Example:
-		// https://github.com/aviator-co/av/compare/master...my-fancy-feature?quick_pull=1
-		_, _ = fmt.Printf(
-			"%s/%s/compare/%s...%s?quick_pull=1\n",
-			config.GitHub.BaseUrl,
-			origin.RepoSlug,
-			prBaseBranch,
-			currentBranch,
-		)
+		client, err := gh.NewClient(config.GitHub.Token)
+		if err != nil {
+			return err
+		}
+
+		body := prCreateFlags.Body
+		// Special case: ready body from stdin
+		if body == "-" {
```

Comment:
so empty body in params defaults to a `-`? Shouldn't this be an empty string

Reply:
No, default *is* an empty string, where we just make the body empty. In the future, I want to automatically set it based on the first commit message. But as a special case, you can read from stdin.

So:
```
# Create PR with empty body:
av pr create -t 'Hello, world!'

# Create PR with body from stdin
echo 'This is a cool PR!' | av pr create -t 'Hello, world!' -b -

# Or with heredoc syntax
av pr create -t 'Hello, world!' -b - <<EOF
This is a cool PR!
EOF
```